### PR TITLE
RWA-837 add new WA FTs for additional properties 

### PR DESCRIPTION
**Pull request by serdaroguzhanakin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-837

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerCFTTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+        Response initiationResponse = restApiActions.post(
+            ENDPOINT_BEING_TESTED,
+            taskId,
+            req,
+            caseworkerCredentials.getHeaders()
+        );
+
+        //Note: this is the TaskResource.class
+        initiationResponse.prettyPrint();
```
Comment (dharmendrak) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerCFTTest.java:75]: please remove after testing unless other reason

**Review by dharmendrak — DISMISSED** [review]

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerCFTTest.java (hunk 1 above)
Comment (BenNzewi) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerCFTTest.java:75]: can we remove this as per previous comment?

**Review by BenNzewi — COMMENTED** [review]
Can we update the relevant PACT tests for getting tasks to show support fo additional properties?

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerCFTTest.java (hunk 1 above)
Comment (serdaroguzhanakin) [reply src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerCFTTest.java:75]: done

**Review by BenNzewi — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**